app/api/v1/responses/wallet_response.py

```
@@ -0,0 +1,36 @@
+class WalletItemResponse(BaseModel):
+    currency_id: int
+    amount: int
+    currency_name: dict[str, str]
+    currency_icon: Optional[str]
+    currency_description: dict[str, Optional[str]]
+
+    @classmethod
+    def map_response(cls, wallet_entry: dict, currency: dict) -> Self:
+        # Validate required keys
+        if 'currency_id' not in wallet_entry or 'amount' not in wallet_entry:
+            raise ValueError(f"Invalid wallet_entry structure: {wallet_entry}")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)